tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -348,14 +394,20 @@ Replace data files for amendment
     user chooses file   id:dataFileUploadForm-dataFile       ${FILES_DIR}dates.csv
     user chooses file   id:dataFileUploadForm-metadataFile   ${FILES_DIR}dates.meta.csv
     user clicks button  Upload data files
-    user checks headed table body row cell contains  Status          2  Complete   wait=180
+    user opens details dropdown  Footnote 2 seven filters
+    user clicks button  Delete footnote
+    user clicks button  Confirm
+    user reloads page
```

Comment:
I'd add a comment at the end of this line linking to the bug this works around - `# EES-1442`